recipes/get-type-of-object.md

```diff
@@ -0,0 +1,40 @@
+# Get type of object
+
+## Problem
+
+You need a procedure that will return the type of the object as a string.
+
+## Solution
+
+```Scheme
+(define (type obj)
+  (cond ((number? obj) "number")
+        ((pair? obj) "pair")
+        ((null? obj) "nil")
+        ((string? obj) "string")
+        ((symbol? obj) "symbol")
+        ((vector? obj) "vector")
+        ((procedure? obj) "procedure")
+        ((port? obj)
+         (cond ((input-port? obj) "input-port")
+               ((output-port? obj) "output-port")
+               (else "unknown-port")))
+        ((eof-object? obj) "eof")
+        ((char? obj) "character")
+        ((boolean? obj) "boolean")))
```

Comment:
Could return symbols instead of strings? Common Lisp uses symbols to name types, and if we made a type system for Scheme, it would most likely use symbols too.

Reply:
Otherwise LGTM